azkaban-web-server/src/test/java/azkaban/flowtrigger/TriggerInstanceProcessorTest.java

```diff
@@ -82,11 +106,22 @@ public void testProcessSucceed() throws ExecutorManagerException {
   }
 
   @Test
-  public void testProcessTermination() throws ExecutorManagerException, InterruptedException {
+  public void testProcessTermination()
+      throws ExecutorManagerException, InterruptedException, IOException {
     final TriggerInstance triggerInstance = createTriggerInstance();
     processor.processTermination(triggerInstance);
     sleep(1000);
```

Comment:
Not in the scope of this PR, but sleep should be replaced with some dynamic wait to make unit tests run fast.

Reply:
thanks